apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -0,0 +1,30 @@
+import { Post } from 'apps/mull-ui/src/generated/graphql';
+import { avatarUrl } from 'apps/mull-ui/src/utilities';
```

Comment:
You should not use relative imports here. Use this instead `../../generated/graphql`

Reply:
hmmm auto-import in vscode doesn't actually use relative path, I will find a way to fix this